src/webhook/webhook.service.ts

```diff
@@ -13,16 +13,25 @@ export class WebhookService {
     private readonly prismaService: PrismaService,
   ) {}
 
-  async sendWebhookWithRecipe(recipe: RecipeResponse): Promise<void> {
-    const webhookURL = process.env.WEBHOOK_URL;
+  async createRecipeWebhook(recipe: RecipeResponse): Promise<void> {
+    const webhooks = await this.fetchAllWebhooks();
 
-    try {
-      await lastValueFrom(this.httpService.post(webhookURL, recipe));
-    } catch (error) {
-      throw new Error('There was an error while sending request');
+    const createRecipeWebhook = webhooks.filter((webhook) => {
+      if (webhook.name === process.env.CREATE_RECIPE_WEBHOOK_NAME) {
```

Comment:
you should not fetch all webhooks and filter them out based on the env value. There is no need to do that. 
If you have fixed set of supported (existing webhooks) you could create and enum for that. After all webbhook name should not be changed by the user.

You should fetch the webhook by it's name directly from the db (findOne) and even better: find one where name === 'x' and enabled === true

Reply:
Side note about filter method. Please reffer to this doc
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

you could do it in one line of code:
const webhook = webhooks.filter((webhook) => webhook.name ===  'someName');